source/synthDrivers/espeak.py

```diff
@@ -25,6 +23,15 @@
 	PhonemeCommand,
 )
 
+
+# A mapping of commonly used language codes to eSpeak languages.
+# These are used when eSpeak doesn't support a given language code
+# but a default alias is appropriate
+_defaultLangToLocaleMappings = {
+	"en": "en-gb",
+}

```

Comment:
Other languages are handled by `_anyLocaleMatchingLang`.
I think this mapping should be expanding as well.

Reply:
I think that we should make this an explicit mapping. IE for every unique two letter language code, have a full locale mapping here. Just to make it clear and easy to determine what the expected behavior should be. This will also protect us against changing behaviour due to order changes in available languages.

Reply:
I don't think this possible as there are not necessarily clear default mappings. It's also not obvious which language codes need a mapping to an eSpeak language.

Reply:
A unit test - 097e434145d7ae2ad00e5043ed9a68f524ca4482, has been added to ensure the mappings are comprehensive